index.md

```diff
@@ -105,6 +157,7 @@ measures.
       - **&lt;readgroup>**: Contains QC metrics and preprocessed FastQ files,
       in case preprocessing was necessary.
 - **multisample.vcf.gz**: A multisample VCF file with the variantcalling
+- **multiqc**: Contains the multiqc report.
 results.
 
 ## About
```

Comment:
This header should be removed and the contents of this section should be moved to above usage.